extensions/s3/src/test/java/io/deephaven/extensions/s3/testlib/S3SeekableChannelTestSetup.java

```diff
@@ -71,14 +72,15 @@ protected final void putObject(String key, AsyncRequestBody body)
     }
 
     protected final SeekableChannelsProvider providerImpl() {
-        final S3SeekableChannelProviderPlugin plugin = new S3SeekableChannelProviderPlugin();
-        final S3Instructions instructions = s3Instructions(S3Instructions.builder()).build();
-        return plugin.createProvider(SCHEME, instructions);
+        return providerImpl(null);
```

Comment:
This might simplify your implmenetation of `providerImpl(s3InstructionsBuilder)`
```suggestion
        return providerImpl(S3Instructions.builder());
```